src/scenes/ModulesRouter/ModuleViewTitle.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Box, Typography } from '@mui/material';
+
+import { Module } from 'src/types/Module';
+
+type ModulePageTitleProps = {
+  module: Module | undefined;
+};
+
+const ModuleViewTitle = ({ module }: ModulePageTitleProps) => {
+  return (
+    <Box alignItems='center' display='flex' flexDirection='row' flexWrap='wrap' marginY='24px' width='100%'>
+      <Typography fontSize={'24px'} lineHeight={'32px'} fontWeight={600} paddingRight='24px' whiteSpace='nowrap'>
+        {module ? 'AND_Climate Action Network-Andromeda' : ''}
```

Comment:
Where should the name in the page title come from, the project? Participant? Something else? (`'AND_Climate Action Network-Andromeda'`)

Reply:
This is the project name! I will make sure to let design and product know to standardize these in the future in our FIgma.